.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  ci:
+    name: >
+      Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
```

Comment:
^ We're not actually running checks - it's a misleading description.

But perhaps we can start running them on GHA?

Reply:
db88816